src/main/java/org/jboss/tools/intellij/openshift/tree/application/ApplicationsRootNode.java

```diff
@@ -74,18 +74,21 @@ public void setLogged(boolean logged) {
         this.logged = logged;
     }
 
-    public CompletableFuture<Odo> initializeOdo() {
-        return OdoCliFactory.getInstance().getOdo(project).whenComplete((odo, err) -> {
-            if (this.odo != null) {
-                this.odo.release();
-            }
-            this.odo = new OdoProjectDecorator(odo, this);
-            loadProjectModel(project);
```

Comment:
this call is still required when loading the root node in applicationTreeStructure class